### PR TITLE
small correction in config.ini

### PR DESCRIPTION
**Pull request by ccesare**
I fixed a small typo in `config.ini`.

### CODE REVIEW

**Review by AnonGit90210 — COMMENTED** [review]
Hi Cesare!
We apologize for the delayed response. We reviewed the changes, but can't find the difference (seems like the exact two lines are deleted and re-pasted, lines 69 and 78). Could you please point out the correction?
Thanks!

File: config.ini [hunk 1]
```diff
@@ -66,7 +66,7 @@ lhs_params=[[[-3, 3], [-3, 3], [-3, 3]], [[-3, 3], [-3, 3], [-3, 3]], true]
 ; the subgroup of postprocessing functions
 postproc_funcs_filter=["safe_inverse", "lambda x: x", "lambda x: x**2", "lambda x: safe_inverse(x**2)", "safe_sqrt", "lambda x: safe_inverse(safe_sqrt(x))"]
 
-; options for hastable_file_opeartion: use, generate, expand
+; options for hastable_file_operation: use, generate, expand
```
Comment (ccesare) [inline config.ini:69]: My fix was to the misspelling of "operation" (it was originally "opeartion"). I now notice that "hastable" should probably be "hashtable," too.

File: config.ini [hunk 2]
```diff
@@ -75,4 +75,4 @@ hashtable_file_operation=use
 hashtable_file=hashtable_general.pkl
 
 ; printing slow convergence cases that we thought should be fast (can be identified by the discriminant)
-print_surprising_nonexp_contfracs=False
\ No newline at end of file
+print_surprising_nonexp_contfracs=False
```
Comment (ccesare) [inline config.ini:78]: This is an artifact of my editor, no doubt. It just removes a trailing newline.

**Comment by ccesare** [comment]
I added some comments, but here is the gist. On line 69, I fixed a misspelling of "operation" (it was originally "opeartion"). Line 78 removes a trailing newline, which I'm sure my editor did by default.

Additionally, "hashtable" is misspelled on line 69 (it reads "hastable"), which I just noticed.

File: config.ini (hunk 1 above)
Comment (AnonGit90210) [reply config.ini:69]: Oh, missed the ar<->ra. Will fix the hastable too. Thanks!

**Pull request merged**